views/profile.handlebars

```diff
@@ -53,4 +53,5 @@
   {{/if}}
 </div>
 
-<script src="../public/js/profile.js"></script>
\ No newline at end of file
+<script src="./js/profile.js"></script>
+{{!-- public is being treated as root folder --}}
```

Comment:
here, the path should be: ="/js/profile.js" - i also fixed this in my session, among a few other handlebars' incorrect paths..